contributors/devel/owners.md

```diff
@@ -1,100 +1,228 @@
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/docs/devel/owners.md?pixel)]()
-<!-- END MUNGE: GENERATED_ANALYTICS -->
+## Code Review Process
+
+This is a simplified description of our [full PR testing and merge
+workflow](https://github.com/kubernetes/community/blob/master/contributors/devel/pull-requests.md#the-testing-and-merge-workflow)
+that conveniently forgets about the existence of tests, to focus solely on the roles driven by
+OWNERS files.
+
+- The **author** submits a PR
+- Phase 0: Automation determines **reviewers** and **approvers** for the PR
+  - Determine the set of OWNERS files nearest to the code being changed
+  - Choose two **reviewers**, and assign them to the PR (choose more than one to avoid assigning to
+    an inactive / unavailable reviewer)
+  - Choose suggested **approvers**, one from each OWNERS file, and list them in a comment on the PR
+- Phase 1: Humans review the PR
+  - **Reviewers** look for general code quality, correctness, sane software engineering, style, etc.
+  - The PR **author** cannot be their own **reviewer**
+  - If the code changes look good to them, a **reviewer** types `/lgtm` in a PR comment or review;
+    if they change their mind, they `/lgtm cancel`
+  - Once a **reviewer** has `/lgtm`'ed, `prow` ([@k8s-ci-robot](https://github.com/k8s-ci-robot/))
+    applies an `lgtm` label to the PR
+- Phase 2: Humans approve the PR
+  - The PR **author** `/assign`'s all suggested **approvers** to the PR, and optionally notifies
+    them (eg: "pinging @foo for approval")
+  - The PR **author** can be their own **approver** (assuming they are listed in the appropriate
+    OWNERS files)
+  - **Approvers** look for holistic acceptance criteria, including dependencies with other features,
+    forwards/backwards compatibility, API and flag definitions, etc
+  - If the code changes look good to them, an **approver** types `/approve` in a PR comment or
+    review; if they change their mind, they `/approve cancel`
+  - `mungegithub` ([@k8s-merge-robot](https://github.com/k8s-merge-robot/)) updates its comment in
+    the PR to indicate which **approvers** still need to approve
+  - Once all **approvers** (one from each of the previously identified OWNERS files) have approved,
+    `mungegithub` ([@k8s-merge-robot](https://github.com/k8s-merge-robot/)) applies an `approved`
+label
+- Phase 3: Automation merges the PR
+  - All required labels are present (eg: `lgtm`, `approved`)
+  - All required status checks for the PR are verified as green
+  - The PR is merged
+
+## Quirks of the Process
+
+There are a number of behaviors we've observed that while _possible_ are discouraged, as they go
+against the intent of this review process.  Some of these could be prevented in the future, but this
+is the state of today.
+
+- An **approver**'s `/lgtm` is simultaneously interpreted as an `/approve`.
+  - This goes against the idea of having two sets of eyes on a PR, and can be confusing to an
```

Comment:
Another quirk would be when the approver selected by mungebot is also one of the two reviewers selected. I don't know if there is a better way to avoid this.